week2/index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>LinkBrary</title>
+  <link rel="stylesheet" href="index.css" />
+  <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/variable/pretendardvariable-gov-dynamic-subset.css" />
+  <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/static/pretendard-gov.css" />
+  <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/variable/pretendardvariable-gov.css" />
+</head>
+<body>
+  <header>
+    <div class="fixsed_always">
+      <h1>
+        <a id="logo" href="/"><img src="img/logo.svg"></a>
+      </h1>
+      <a class="use_site" id="signin" href="/signin.html">로그인</a>
+    </div>
+    <article>
+      <h2 class="title">
+        <span class="point">세상의 모든 정보</span>를<br>
+        쉽게 저장하고 관리해 보세요
+      </h2>
+    <a class="use_site" id="sigup" href="/sigup.html">링크 추가하기</a>
+      <div class="search_page_box"></div>
+  </header>
+ 
+  <main>
+    <article class="information chapt1">
+      <section>
+        <h3 class="little_title">
+          <span class="point">원하는 링크</span>를<br>
+          저장하세요
+        </h3>
+        <p>
+          나중에 읽고 싶은 글, 다시보고 싶은 영상,<br>
+          사고싶은 옷, 기억하고 싶은 모든 것을<br>
+          한 공간에 저장하세요
+        </p> 
+      </section>
+      <img src="img/_img1.png">
+    </article>
+
+    <article class="information chapt2 side">
+      <section>
+        <h3 class="little_title">
+          링크를 폴더로<br>
+          <span class="point">관리</span>하세요
+        </h3>
+        <p>
+          나만의 폴더를 무제한으로 만들고<br>
+          다양하게 활용할 수 있습니다.<br>
+        </p> 
+      </section>
+      <img src="img/img2.png">
+    </article>
+
+    <article class="information chapt3">
+      <section>
+        <h3 class="little_title">
+          저장한 링크를 <br>
+          <span class="point">공유</span>해 보세요
+        </h3>
+        <p>
+          여러 링크를 폴더에 담고 공유할 수 있습니다.<br>
+          가족, 친구, 동료들에게 쉽고 빠르게 링크를<br>
+          공유해 보세요.
+        </p> 
+      </section>
+      <img src="img/img3.png">
+
+    </article>
+
+    <article class="information chapt4 side">
+      <section>
+        <h3 class="little_title">
+          저장한 링크를<br>
+          <span class="point">검색</span>해 보세요
+        </h3>
+        <p>
+         중요한 정보들을 검색으로 쉽게 찾아보세요.
+        </p> 
+      </section>
+      <img src="img/_img4.png">
+    </article>
+  </main>
+
+  <footer>
+    <article  id="underbar">
+      <section id="small_logo">
+        ⓒcodeit - 2023
```

Comment:
요런 방식도 있으니 참고해주세요
https://developer.mozilla.org/en-US/docs/Glossary/Entity

```suggestion
        &copy;codeit - 2023
```